week2/week2_day5_book_exercises.Rmd

```diff
@@ -0,0 +1,87 @@
+---
+title: "week2_day5_book_exercises"
+author: "Christopher Esquivel"
+date: '2022-06-10'
+output:
+  html_document: default
+  pdf_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+##
+Chapter 5 exercises
+5.20
+a) The equation is y = 0.607975x + 105.2963
+```{r}
+avg_shoulder_girth <- 108.20
+avg_height <- 171.14
+sd_shoulder_girth <- 10.37
+sd_height <- 9.41
+R <- 0.67
+
+b1 <- (sd_height / sd_shoulder_girth) * R
+b0 <- 105.2963
+
+```
+b) For the slope, for every increase of 10 cm in shoulder girth, we expect an increase in height of
+6.07675 cm on average. The intercept describes the average height (105.2963 cm) when the shoulder girth is a small value.
+
+c) R-squared is 0.4489 or 45%. R-squared is the strength of a linear fit.

```

Comment:
"Strength of a linear fit" is a little too broad of an explanation of what R^2 is.